src/api/appSetting.ts

```diff
@@ -26,7 +27,7 @@ export const postAppSetting = (
   args: ApiData & {
     body: unknown;
   },
-) => {
+): Promise<AppSetting> => {
   const { token, itemId, apiHost, body } = args;
   return axios
     .post(`${apiHost}/${buildPostAppSettingRoute({ itemId })}`, body, {
```

Comment:
Actually I noticed it was possible to set the type on axios, I think it's better.

```suggestion
) => {
  const { token, itemId, apiHost, body } = args;
  return axios
    .post<AppSetting>(`${apiHost}/${buildPostAppSettingRoute({ itemId })}`, body, {
```